projects/plugins/jetpack/extensions/blocks/paywall/index.js

```diff
@@ -27,7 +27,7 @@ export const title = __( 'Paywall', 'jetpack' );
 export const settings = {
 	title,
 	description: __(
-		'Add a paywall block to make part of your content exclusive to paid subscribers.',
+		'Content before this block will be shown in the excerpt on your archives page.',
```

Comment:
I'll update this also in [`block.json`](https://github.com/Automattic/jetpack/edit/af6bbd095b61806f8dcd806ce92d6c8acf7eee76/projects/plugins/jetpack/extensions/blocks/paywall/block.json) that we now have. I don't think it's used yet since blocks are being migrated.